tests/integration-tests/tests/common/schedulers_common.py

```diff
@@ -405,9 +405,28 @@ def _submit_batch_job(
         else:
             return self._remote_command_executor.run_remote_command(submission_command, raise_on_error=raise_on_error)
 
+    def _dump_job_output(self, job_info):
+        match = re.match(r".*StdErr=(.*)?\sStdIn=(.*)?\sStdOut=(.*)?\sPower.*", job_info)
+        if match:
+            stderr = match.group(1)
+            stdout = match.group(3)
+            if stdout == stderr:
+                command = f'echo "stderr/stdout:" && cat {stderr}'
+            else:
+                command = f'echo "stderr:" && cat {stderr} && echo "stdout:" && cat {stdout}'
+            result = self._remote_command_executor.run_remote_command(command)
+
+            return result.stdout
+
+        return ""
+
     def assert_job_succeeded(self, job_id, children_number=0):  # noqa: D102
         result = self._remote_command_executor.run_remote_command("scontrol show jobs -o {0}".format(job_id))
-        assert_that(result.stdout).contains("JobState=COMPLETED")
+        try:
+            assert_that(result.stdout).contains("JobState=COMPLETED")
+        except AssertionError as error:
+            error.args += (self._dump_job_output(result.stdout),)
```

Comment:
optional: we could also simply logging.error

Reply:
done, use logging instead